src/shared/ui/survey/Slider/SurveySlider.tsx

```diff
@@ -72,7 +78,7 @@ const SurveySlider: FC<SliderProps & AccessibilityProps> = ({
 
       <XStack mt="$2" jc="space-between">
         <YStack maxWidth="30%" ai="center">
-          {leftImageUrl && (
+          {leftImageUrl ? (
```

Comment:
What if we wrap the if-expression into another Box element with the size of 44x44?